### PR TITLE
add restore from ufvk test

### PR DESCRIPTION
**Pull request by Oscar-Pepper**

### CODE REVIEW

File: android/app/src/androidTest/java/org/ZingoLabs/Zingo/RestoreFromUfvk.kt [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            assertThat(true).isEqualTo(false)
+        }
+        
+        val server = "https://mainnet.lightwalletd.com:9067"
+        val chainhint = "main"
+        val ufvk = "uview1vmle9qzgsmzn8qskg06cjjt406eq3euvkn8un3t6sk7fjd72qp72guyskktr2gvgcpmd99qgquxdw7s54v0dfxn3degzhxp9gmvpve6vlvv459pxfre05v6l47aqx47rgh86t6n77svstd8ff6d8cwtn98uq66k6u5jqrpyz0pqflkppfq3djscrd2acnc7ymkd6fssk0t4rh2ynaux7z4ylt38jqgjhnu88h2jz8qwd3t5dwsc3ycvsea6grs4zg76r9vw48r9zvlphnpfsgc5eqpum7hghdm2eguw4h6n9m8rhuzh0qdc24z4z5ftcv6xxvvs3yrugea23xlys3f9qv0fenh0xp0hej8enlr82esl54hn27d6432kygwqx700ez84e72f03vtmece73dftpjvv3v7w65yaz2rjwmarxjzrnn02u5kx4p7a42k5lxgcgqgwjyp9w6x24ccm0dvlf4637ss6l3xmuv2strza60c5k5uasdcn"
+        val birthday = "419200"
+        val datadir = MainApplication.getAppContext()!!.filesDir.path
+
+        // RustFFI.initfromufvk(server, ufvk, birthday, datadir, chainhint)
```
Comment (juanky201271) [inline android/app/src/androidTest/java/org/ZingoLabs/Zingo/RestoreFromUfvk.kt]: If you are using this line... you need to know what is the response of this function, normally will be this: `Error: This wallet is watch-only.`, in this case, everything is OK, but here could be different result.

File: android/app/src/androidTest/java/org/ZingoLabs/Zingo/RestoreFromUfvk.kt [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+        val resp = RustFFI.execute("addresses", "")
+        val addresses: List<Addresses> = jacksonObjectMapper().readValue(resp)
```
Comment (juanky201271) [inline android/app/src/androidTest/java/org/ZingoLabs/Zingo/RestoreFromUfvk.kt]: This line could fail if the `addresses` command have any error... in that case the `resp` string begins with `Error: ...................................`. You need to control this.

**Review by juanky201271 — CHANGES_REQUESTED** [review]

File: android/app/src/androidTest/java/org/ZingoLabs/Zingo/RestoreFromUfvk.kt (hunk 1 above)
Comment (Oscar-Pepper) [reply android/app/src/androidTest/java/org/ZingoLabs/Zingo/RestoreFromUfvk.kt]: seems strange that the happy path returns an "Error: ... " ?

File: android/app/src/androidTest/java/org/ZingoLabs/Zingo/RestoreFromUfvk.kt (hunk 2 above)
Comment (Oscar-Pepper) [reply android/app/src/androidTest/java/org/ZingoLabs/Zingo/RestoreFromUfvk.kt]: good advice thanks... im removing from this test but will open a PR to fix the executeAddresses test

**Comment by Oscar-Pepper** [comment]
continued in #412

**Pull request closed without merging**